webpack/assets/javascripts/react_app/components/HostDetails/index.js

```diff
@@ -216,7 +216,11 @@ const HostDetails = ({
               <Text ouiaId="date-text" component={TextVariants.span}>
                 <RelativeDateTime date={response.created_at} defaultValue="N/A">
                   {date =>
-                    sprintf(__('Created %s by %s'), date, response.owner_name)
+                    sprintf(
+                      __('Created %s owned by %s'),
+                      date,
+                      response.owner_name
```

Comment:
I believe that the original issue was that we don't display the information about the user who created the host.
The owner of the host and the user who created the host are not the same users sometimes.
One option to retrieve that data is possible through Audits, e.g:
```rb
Audit.where(id: host.audit_ids, action: 'create').first.username
```

another option could be to save that data as `user_id` or `created_by_user` on the host model on creation, so it will be easier to retrieve that data.

so later in the UI I would expect something like:
```js
sprintf(__('Created %s by %s'), date, response.created_by_user)
```

Reply:
:dart:

Reply:
That sounds correct, though we should also consider cases where the information is unavailable. For example, what if the user is removed? Perhaps audit logs have been pruned. The host may also have been created via automation (sending in facts for example) in which case I think the active user is the admin user. I'm not sure we can properly distinguish that.

That's why I think the UI should somehow deal with no (known) creator. For example, omitting the field, showing "unavailable" or something else.

Reply:
Thanks for the response @ekohl!